dhcpv4/dhcpv4.go

```diff
@@ -37,6 +38,33 @@ type DHCPv4 struct {
 	options        []Option
 }
 
+// interfaceV4Addr obtains the currently-configured IPv4 address for iface.
```

Comment:
this function actually gets the *first* IPv4 address configured for the interface. I'd rather return a list of `net.IP` instead, and leave to the caller to pick the first one